yarn.lock

```diff
@@ -10935,9 +10891,14 @@ tslib@1.10.0:
   integrity sha512-qOebF53frne81cf0S9B41ByenJ3/IuH8yJKngAX35CmiZySA0khhkovshKK+jGCaMnVomla7gVlIcc3EvKPbTQ==
 
 tslib@^1.10.0, tslib@^1.8.1, tslib@^1.9.0, tslib@^1.9.3:
-  version "1.11.1"
-  resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.11.1.tgz#eb15d128827fbee2841549e171f45ed338ac7e35"
-  integrity sha512-aZW88SY8kQbU7gpV19lN24LtXh/yD4ZZg6qieAJDDg+YBsJcSmLGK9QpnUjAKVG/xefmvJGd1WUmfpT/g6AJGA==
+  version "1.13.0"
```

Comment:
multiple versions

Reply:
Likewise to the previous duplicates. I do wonder if we should update to tslib 2.0.0 in the repo here as well though. Thoughts?

Reply:
Any chance that'll break things?

Reply:
It shouldn't, as many others should be using it.

Reply:
Upgraded to tslib 2.0.0 and fixed that dep missing on a number of our various modules.

Note: Packages outside of our modules still rely on tslib 1.x, but that won't impact us.